html/pfappserver/root/static.alt/src/components/pfNetworkGraphTooltipSwitch.vue

```diff
@@ -75,10 +48,52 @@ export default {
       default: () => { return {} }
     }
   },
-  computed: {
-    hasLocationLogProperties () {
-      return 'locationlog' in this.properties && Object.keys(this.properties.locationlog).filter(key => this.properties.locationlog[key]).length > 0
+  data () {
+    return {
+      switche: false,
+      isLoading: false,
+      isError: false
+    }
+  },
+  methods: {
+    init () {
+      this.isLoading = true
+      apiCall.getQuiet(`config/switch/${this.id}`).then(response => {
```

Comment:
Could we avoid an API call when `this.id` is `unknown`?

Reply:
I guess we could benefit from a vuex store here to do some caching and speedup the tooltips.